homeassistant/components/tellstick/__init__.py

```diff
@@ -179,7 +179,10 @@ def __init__(self, tellcore_device, signal_repetitions):
 
         # Look up our corresponding tellcore device
         self._tellcore_device = tellcore_device
-        self._name = tellcore_device.name
+        self._attr_assumed_state = True
+        self._attr_name = tellcore_device.name
+        self._attr_should_poll = False
+        self._attr_unique_id = tellcore_device.id
```

Comment:
I think this is not correct. If a device provides multiple entities (e.g., multiple sensors), a unique ID collision would occur.

Reply:
I think a device that provides multiple sensors will have multiple id's. But I don't know about these devices. Every device (sensor, switch, ...) has an internal id (numeric) as it is configured in the tellstick configuration.

Reply:
A device in this context is a sensor or a switch, not the tellstick itself.

I have a 433Mhz remote control with 4 on/off buttons, and those are 4 switches with 4 id's.

Reply:
Alright, thanks for clarifying 👍